driver-core/src/main/com/mongodb/connection/TransportSettings.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.connection;
+
+import com.mongodb.connection.netty.NettySettings;
+
+/**
+ * Transport settings for the driver.
+ *
+ * @since 4.11
+ */
+public abstract class TransportSettings {
+    /**
+     * A builder for NettySettings.
+     *
+     * @return a builder for NettySettings
+     */
+    public static NettySettings.Builder nettyBuilder() {
```

Comment:
I added this for discoverability, but it duplicates `NettySettings.builder()`.  Should we have both public, only this one, or only `NettySettings.builder()?

Reply:
I prefer having these only on `TransportSettings`. If we ever need them on subtypes, which I doubt, we will be able to add the methods.